src/sentry/monitors/models.py

```diff
@@ -55,18 +55,19 @@ def get_next_schedule(last_checkin, schedule_type, schedule):
     return next_schedule
 
 
-def get_monitor_context(monitor):
-    config = monitor.config.copy()
+def get_monitor_environment_context(monitor_environment):
+    config = monitor_environment.monitor.config.copy()
     if "schedule_type" in config:
-        config["schedule_type"] = monitor.get_schedule_type_display()
+        config["schedule_type"] = monitor_environment.monitor.get_schedule_type_display()
 
     return {
-        "id": str(monitor.guid),
-        "slug": monitor.slug,
-        "name": monitor.name,
-        "config": monitor.config,
-        "status": monitor.get_status_display(),
-        "type": monitor.get_type_display(),
+        "id": str(monitor_environment.monitor.guid),
+        "slug": monitor_environment.monitor.slug,
+        "name": monitor_environment.monitor.name,
+        "environment": monitor_environment.environment.name,
```

Comment:
do we want to include `environment` in the context here? or just in the tags?

Reply:
It should just be in the event